### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to version 0.30.0*

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "multi_instance": false,
     "services": [
         "nginx",
-        "php7.0-fpm"
+        "php7.3-fpm"
```
Comment (alexAubin) [inline manifest.json:22]: If you require php7.3-fpm we should require yunohost >= 4.0 ?

**Review by alexAubin — APPROVED** [review]
LGTM apart from version requirement :+1:

**Comment by alexAubin** [comment]
!testme

**Pull request merged**